src/records/RecHttp.cc

```diff
@@ -150,7 +151,7 @@ RecHttpLoadIpAddrsFromConfVar(const char *value_name, swoc::IPRangeSet &addrs)
 
   if (REC_ERR_OKAY == RecGetRecordString(value_name, value, sizeof(value))) {
     Debug("config", "RecHttpLoadIpAddrsFromConfVar: parsing the name [%s] and value [%s]", value_name, value);
-    swoc::TextView text(value);
+    swoc::TextView text(value, std::strlen(value));
```

Comment:
Probably better to favor `strnlen(value, 1024)`.

Reply:
Can we use this even though it's part of the C standard and not part of the C++ standard? https://en.cppreference.com/w/cpp/string/byte/strlen

The rest of records uses `strlen(str)` after calling `RecGetRecordString` and this is safe because it's guaranteed to always null-terminate the buffer.

Reply:
True. `strlen` should be fine.